test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPushCommandTest.cs

```diff
@@ -336,13 +338,6 @@ public void PushCommand_PushToServerBasicAuth()
                     using (var server = new MockServer())
                     {
                         server.Listener.AuthenticationSchemes = AuthenticationSchemes.Basic;
-                        server.Get.Add("/nuget", r =>
```

Comment:
FYI, like i commented out on the redirect tests, it was agreed we don't need to deal with redirect support.